dev/Deployment/DeploymentManager.cpp

```diff
@@ -25,6 +29,12 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         return Initialize(GetCurrentFrameworkPackageFullName());
     }
 
+    winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentResult DeploymentManager::Initialize(winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentInitializeOptions const& deploymentInitializeOptions)
+    {
+        FAIL_FAST_HR_IF(HRESULT_FROM_WIN32(APPMODEL_ERROR_NO_PACKAGE), !AppModel::Identity::IsPackagedProcess());
```

Comment:
BTW DeploymentManager's documented as not supporting unpackaged?

We should lift that restriction. Packaged Store-distributed apps _need_ this API but (20-20 hindsight) it's useful (and viable) for _everyone_ for _repair_ scenarios if nothing else. Also, given DynDep changes for elevation as long as the DDLM package(s) are registered this could be used by unpackaged apps as a way to install Main+Singleton.

Please file a vFuture issue to lift the current restriction.

Reply:
https://task.ms/38653726